systemd/step-ca.service

```diff
@@ -0,0 +1,52 @@
+[Unit]
+Description=step-ca service
+Documentation=https://smallstep.com/docs/step-ca
+Documentation=https://smallstep.com/docs/step-ca/certificate-authority-server-production
+After=network-online.target
+Wants=network-online.target
+StartLimitIntervalSec=30
+StartLimitBurst=3
+ConditionFileNotEmpty=/etc/step-ca/config/ca.json
+ConditionFileNotEmpty=/etc/step-ca/password.txt
+
+[Service]
+Type=simple
+User=step
+Group=step
+Environment=STEPPATH=/etc/step-ca
+WorkingDirectory=/etc/step-ca
+ExecStart=/usr/local/bin/step-ca config/ca.json --password-file password.txt
+ExecReload=/bin/kill --signal HUP $MAINPID
+Restart=on-failure
+RestartSec=5
+TimeoutStopSec=30
+StartLimitInterval=30
+StartLimitBurst=3
+
+; Process capabilities & privileges
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+SecureBits=keep-caps
+NoNewPrivileges=yes
+
+; Sandboxing
+ProtectSystem=full
+RestrictNamespaces=true
```

Comment:
Is there a reason there is no `ProtectHome=true`? https://github.com/smallstep/docs/blob/master/step-ca/certificate-authority-server-production.mdx#running-step-ca-as-a-daemon contained it and according to https://www.freedesktop.org/software/systemd/man/systemd.exec.html this should do: "If true, the directories /home/, /root, and /run/user are made inaccessible and empty for processes invoked by this unit."

Reply:
This should be there, good call.